src/Controller/RefileRequestController.php

```diff
@@ -110,20 +114,29 @@ public function createRefileRequest()
             );
 
             // Track status issues for the database for NYPL only.
-            $statusFlag = true;
+            $statusFlag = false;
+            $afMessage = null;
+            $sip2Response = null;
 
             APILogger::addDebug('Received SIP2 message', $result);
 
             // Log a failed SIP2 status change to AVAILABLE without terminating the request prematurely.
-            if ($result['fixed']['Alert'] == 'Y') {
+            if ($result['fixed']['Alert'] == 'N' && $result['fixed']['Ok'] == '1') {
```

Comment:
We may need to start documenting all of the cases for these SIP2 responses. Definitions of 'Alert,' 'OK,' etc. and their values. I'm a bit fuzzy on what an 'OK' = '0' and 'Alert' = 'N' scenario would be where we flag that as a true status. I think I did this in reverse purposely since the 'Alert' = 'Y' condition seemed to be more consistent as to not trap false negatives. Seems I lacked adding documentation on this for this project. 

Perhaps you and I can work on adding more information to the Wiki or to another supplemental document about our experience with SIP2.

Reply:
@gkallenberg That's a great idea, I was thinking about it in #20, but not sure if I had the success status all flushed out. Adjusting the logic is trivial code changes, let's do this together and get this right.